### PR TITLE
DBZ-701 Support for arrays with large OID

### PR DESCRIPTION
**Pull request by jpechane**
https://issues.jboss.org/browse/DBZ-701

### CODE REVIEW

File: debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresConnection.java [hunk 1]
```diff
@@ -328,7 +328,7 @@ private static TypeRegistry initTypeRegistry(Connection db, Map<String, Integer>
                                 oid,
                                 nameToJdbc.get(rs.getString("name")),
                                 typeInfo,
-                                typeRegistryBuilder.get(rs.getInt("element"))
+                                typeRegistryBuilder.get((int)rs.getLong("element"))
```
Comment (gunnarmorling) [inline debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresConnection.java]: A couple of lines above (in two places) we also get oids via `getInt()`. Seems we should change those to that pattern as well?

Comment (jpechane) [reply debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresConnection.java]: @gunnarmorling Are you sure? The other `getInt`s were already replaced in previous PR. There is no `getInt` string in https://github.com/jpechane/debezium/blob/e8d5084fbc74739bc02df31439dbede6e7b07020/debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresConnection.java

Comment (gunnarmorling) [reply debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresConnection.java]: True that. I was looking at my (old) local check-out. All good. Thanks!

**Comment by gunnarmorling** [comment]
Applied. Thx!

**Pull request merged**